### PR TITLE
[NEAT-487] Aggegated view of issues

### PR DESCRIPTION
**Pull request by doctrino**
image

### CODE REVIEW

File: cognite/neat/_constants.py [hunk 1]
```diff
@@ -5,6 +5,17 @@
 
 from cognite import neat
 
+IN_NOTEBOOK = True
```
Comment (nikokaoja) [inline cognite/neat/_constants.py:8]: can you wrap it in small method ? that you call ?

File: cognite/neat/_session/_inspect.py [hunk 2]
```diff
@@ -8,8 +14,76 @@
...
+    def __call__(
+        self,
+        search: str | None = None,
+        return_dataframe: Literal[True] = (False if IN_NOTEBOOK else True),  # type: ignore[assignment]
+    ) -> pd.DataFrame: ...
+
+    @overload
+    def __call__(
+        self,
+        search: str | None = None,
```
Comment (nikokaoja) [inline cognite/neat/_session/_inspect.py:42]: nice feature,

File: cognite/neat/_session/_inspect.py [hunk 3]
```diff
@@ -8,8 +14,76 @@
...
+        return_dataframe: bool = (False if IN_NOTEBOOK else True),  # type: ignore[assignment]
+    ) -> pd.DataFrame | None:
+        """Returns the issues of the current data model."""
+        issues = self._state.last_issues
+        if not issues:
+            self._print("No issues found.")
+
+        if issues and search is not None:
+            unique_types = {type(issue).__name__ for issue in issues}
+            closest_match = set(difflib.get_close_matches(search, unique_types))
```
Comment (nikokaoja) [inline cognite/neat/_session/_inspect.py:58]: will this return partial matches ? what about fuzzy instead ? https://github.com/seatgeek/thefuzz or using cognite sdk entity matching ?

**Review by nikokaoja — APPROVED** [review]

File: cognite/neat/_session/_inspect.py (hunk 3 above)
Comment (doctrino) [reply cognite/neat/_session/_inspect.py:58]: This is a very simple way of using the standard library without any extra dependencies to get a slightly better search experience. Suggest we avoid any extra dependency

**Pull request merged**